src/main/java/com/conveyal/datatools/editor/datastore/FeedTx.java

```diff
@@ -124,6 +122,10 @@ public FeedTx(DB tx, boolean buildSecondaryIndices) {
 //        editedSinceSnapshot = tx.getAtomicBoolean("editedSinceSnapshot") == null ? tx.createAtomicBoolean("editedSinceSnapshot", false) : editedSinceSnapshot;
     }
 
+    private static int toGtfsDate (LocalDate date) {
+        return date.getYear() * 10000 + date.getMonthValue() * 100 + date.getDayOfMonth();
```

Comment:
Needs JavaDoc. Also, is there a way to use a date formatting library here?

Reply:
This is deprecated code. It honestly probably doesn't even need to exist in this version. I'm going to create an issue to discuss removing this.

Reply:
Issue is #236.